app/scripts/services/navigate.js

```diff
@@ -70,6 +70,10 @@ angular.module("openshiftConsole")
         return "project/" + encodeURIComponent(projectName) + "/overview";
       },
 
+      toProjectList: function(){
+        $location.path('projects');
```

Comment:
this should always work whether catalog is turned on or not, maybe we should add a redirect from projects to / when the landing page isn't enabled